packages/transaction-controller/src/helpers/GasFeePoller.test.ts

```diff
@@ -758,3 +762,246 @@ describe('updateTransactionGasFees', () => {
     });
   });
 });
+
+describe('applyTransactionGasEstimatesToTransaction', () => {
```

Comment:
Do we need to duplicate all these tests if the logic is the same and we can re-use?